charts/cmc-claim-store/values.aat.template.yaml

```diff
@@ -0,0 +1,10 @@
+claim-store:
+  environment:
+    FRONTEND_BASE_URL: https://moneyclaims.aat.platform.hmcts.net
+    IDAM_S2S_AUTH_URL: http://rpe-service-auth-provider-aat.service.core-compute-aat.internal
+    SEND_LETTER_URL: http://rpe-send-letter-service-aat.service.core-compute-aat.internal
+    RESPOND_TO_CLAIM_URL: https://moneyclaims.aat.platform.hmcts.net/first-contact/start
+    IDAM_API_URL: https://preprod-idamapi.reform.hmcts.net:3511
+    PDF_SERVICE_URL: http://cmc-pdf-service-aat.service.core-compute-aat.internal
+#    CCD_GATEWAY_HOST: http://${SERVICE_NAME}-api-gateway
```

Comment:
does claim store talk to ccd api gateway? doesn't sound right to me

Reply:
oh sorry - can you ignore the commented out bits for now. im working on ccd integration next - will complete with another PR